flink-metrics/flink-metrics-influxdb/src/main/java/org/apache/flink/metrics/influxdb/InfluxdbReporterFactory.java

```diff
@@ -0,0 +1,14 @@
+package org.apache.flink.metrics.influxdb;
```

Comment:
This file is missing the apache license header.

Reply:
OK, I'll do it today

Reply:
@zentol Apache license added